README.md

```diff
@@ -84,8 +84,17 @@ runner.run(collection, {
         // Enable sending of bodies with GET requests (only supported on Node, ignored in the browser)
         sendBodyWithGetRequests: true,
 
-        // Allows restricting IP/host in requests
-        restrictedAddresses: {'192.168.1.1': true}
+        // network related options
+        network: {
+            hosts: { // hosts configuration for dns lookup
```

Comment:
shouldn't this be something related to `lookup`?

Reply:
renaming to `hostLookup`